src/PowerShellEditorServices.Hosting/PowerShellEditorServices.Hosting.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.1;net461</TargetFrameworks>
+    <AssemblyName>Microsoft.PowerShell.EditorServices.Hosting</AssemblyName>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.1' ">
+    <DefineConstants>$(DefineConstants);CoreCLR</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NETStandard.Library" Version="2.0.3" />
```

Comment:
So this *should* mean we get `netstandard.dll` for net461 (and nothing in .NET Core). Basically it's the binding trampoline from netstandard to the concrete runtime.

One thing I've noticed though is that in Core it will keep trying to find `netstandard.dll`. That hasn't lead to any crashes, but I wonder if it signifies that we're doing something wrong. As far as I know, .NET Core 2.0+ has netstandard.dll baked in, so there's no physical DLL to find.